.github/workflows/jekyll.yml

```diff
@@ -0,0 +1,59 @@
+name: Jekyll site CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '2.x'
+    - name: Check dates
+      run: |
+        pip install pytz ciso8601
+        for episode in _episodes/*.md; do
+          date=$(grep 'date:' "$episode" | head -n1 | sed 's/^date: //')
+          if ! python2 -c "import ciso8601; ciso8601.parse_rfc3339('$date');"; then
+            echo "$episode: bad date '$date'"
+            exit 1
+          fi
+        done
+    - name: Check file sizes
+      run: |
+        for episode in _episodes/*.md; do
+          length=$(grep 'length:' "$episode" | head -n1 | awk '{print $2}' | sed 's/"//g')
+          file=$(grep 'file:' "$episode" | head -n1 | sed -e 's/^file: //' -e 's/"//g')
+          size=$(curl --head -s "$file" | grep content-length | awk '{print $2}' | sed 's/\r//')
+          if [ $size -ne $length ]; then
+            echo "$(basename "$episode"): ${length}b (reported) != ${size}b (actual)"
+            exit 1
+          fi
+        done
+    - uses: actions/setup-ruby@v1
```

Comment:
Was just reading https://julienrenaux.fr/2019/12/20/github-actions-security-risk/ which suggests to pin actions to a specific hash rather than a tag. We're not passing in any secrets or anything to the action, but it might be a best practice to consider.

Reply:
We are only using "official" GitHub actions here, not third-party/marketplace ones, so I think it should be fine :)